src/braket/aws/aws_quantum_job.py

```diff
@@ -528,10 +528,21 @@ def __eq__(self, other) -> bool:
     def __hash__(self) -> int:
         return hash(self.arn)
 
+    @staticmethod
+    def _get_device_region(device_arn: str) -> str:
+        try:
+            return device_arn.split(":")[3]
+        except IndexError:
+            raise ValueError(
+                f"Device arn is not a valid format: {device_arn}. Examples of correct arns are: "
+                "arn:aws:braket:::device/quantum-simulator/amazon/sv1 or "
+                "arn:aws:braket:us-west-1::device/qpu/rigetti/Aspen-M-1."
```

Comment:
I'm not sure if we should add specific arns here. Might be better to point to the docs specifying the valid arns: https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html